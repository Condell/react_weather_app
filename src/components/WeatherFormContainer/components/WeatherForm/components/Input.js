import styled from 'styled-components';

const Input = styled.input`
  background-color: white;
  border-radius: 5px;
  margin: .5em;
  padding: .5em;
`;

export default Input;