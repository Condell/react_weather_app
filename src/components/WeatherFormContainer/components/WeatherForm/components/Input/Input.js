import styled from "styled-components";
import React from 'react'

const Input = styled.input`
  background-color: white;
  border-radius: 5px;
  margin: .5em;
  padding: .5em;
`;

export default Input;
