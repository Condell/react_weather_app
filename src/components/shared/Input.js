import React from 'react';
import styled from 'styled-components';

const Input = styled.input`
    background-color: white;
    border-radius: 5px;
    margin: 1em;
    padding: .5em;
`;

export default Input;