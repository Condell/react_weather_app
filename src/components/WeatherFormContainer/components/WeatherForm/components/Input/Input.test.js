import React from "react";
import { shallow } from "enzyme";
import 'jest-styled-components'
import toJson from "enzyme-to-json"

import Input from "./Input";

describe("Input", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Input />);
  });

  it("should render", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});