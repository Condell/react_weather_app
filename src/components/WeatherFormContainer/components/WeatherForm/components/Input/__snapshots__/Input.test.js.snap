// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render 1`] = `
.ijtFOJ {
  background-color: white;
  border-radius: 5px;
  padding: .5em;
  margin: .5em;
}

<input
  className="sc-bdVaJa ijtFOJ"
/>
`;
